Dockerfile

```
@@ -0,0 +1,72 @@
+# FROM jupyter/base-notebook
+# USER root
+# ENV DEBIAN_FRONTEND=noninteractive
+FROM ubuntu:xenial-20161213
```

why don't you just use neurodebian images right away?
```suggestion
FROM neurodebian:xenial
````
and then avoid all the manual curl/add-key below?
Also now in our images we install `eatmydata` and symlink it to apt-get so your docker image would build 2-3 times faster I think ;-)

---

I forgot about this. Good idea. I shall add that.